src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -96,6 +114,12 @@
 						infEWASInst.addAttributeValue(name, homologueId);
 					}
 
+					// If the species is C. elegans, add each matching gene name to the 'name' attribute of the EWAS.
+					if (isCelegans) {
+						for (String wormbaseGeneName : wormbaseGeneNames) {
+							infEWASInst.addAttributeValue(name, wormbaseGeneName);
```

Comment:
Again, we should try to use constants in `ReactomeJavaConstants` to refer to the names of data fields.

Reply:
static imports...